Automattic/vip-go-mu-plugins
# Remove VaultPress from admin menu when not connected

## Description

This PR removes VaultPress from the admin sidebar when it is not activated. 

In VIP Go, VaultPress has to be activated from Jetpack by Customer Success, not by the user. If a user tries to navigate to VaultPress' admin page in VIP Go and activate VaultPress, activation will fail. With this PR changes, that page is only accessible once VaultPress has been activated.

**Before**

![Screen Shot 2021-04-30 at 16 50 26](https://user-images.githubusercontent.com/7188409/116713316-f431d700-a9d4-11eb-9801-e73849b3f790.png)

**After**

![Screen Shot 2021-04-30 at 16 51 30](https://user-images.githubusercontent.com/7188409/116713328-f7c55e00-a9d4-11eb-9041-39fa5ea3082c.png)

## Changelog Description

### Remove VaultPress from admin menu when not active

Preventing the navigation to the VaultPress page if it is not installed. 

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

## Steps to Test

1. Check out PR.
2. Make sure you are in a WP site without a Jetpack connection
3. Go to `wp-admin` 
4. You shouldn't see the VP menu item.
